portfolio/src/main/java/com/google/sps/servlets/Serve.java

```diff
@@ -0,0 +1,23 @@
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import javax.servlet.annotation.WebServlet;
+
+@WebServlet("serve")
+public class Serve extends HttpServlet {
```

Comment:
I don't know what the appropriate name of this class is, but Serve seems nondescript. Maybe `BlobServlet`, `StorageServlet`, `UserDataServlet` or with a little rework this can be the `ImageServlet`.

Reply:
changed the name!